### DISCUSSION TITLE
Dúvida referente ao curso de Mult-Tenant

### DISCUSSION THREAD

**Discussion by RomuloJunges** [balta-io/forum#299] [Tira Dúvidas] [answered]
Na visão de uma API, se eu tiver regras de negócio para diferentes serviços e cliente, como eu poderia tratar para se encaixar no mult-tenant e não ficar aquela quantidade enorme de ifs dentro do método? Exemplo: ter 10 services e todas elas possui regras diferentes para diferentes clientes, como eu faço para carregar todas?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @RomuloJunges, como vai?

Consegue dar uma exemplo mais prático do problema? Citar algum caso real que tenha estes serviços? Assim consigo te ajudar mais, sendo mais assertivo.
